configuration/expose-portal-over-http.adoc

```diff
@@ -0,0 +1,51 @@
+[id="expose-portal-over-http"]
+= Exposing RHACS portal over HTTP
+include::modules/common-attributes.adoc[]
+:context: expose-portal-over-http
+
+toc::[]
+
+[role="_abstract"]
+Enable an unencrypted HTTP server to expose the RHACS portal through ingress controllers, Layer 7 load balancers, Istio, or other solutions.
+
+If you use an ingress controller, Istio, or a Layer 7 load balancer that prefers unencrypted HTTP back ends, you can configure {product-title} to expose the RHACS portal over HTTP.
+Doing this makes the RHACS portal available over a plaintext back end.
+
+[IMPORTANT]
+====
+To expose the RHACS portal over HTTP, you must be using an ingress controller, a Layer 7 load balancer, or Istio to encrypt external traffic with HTTPS.
+It is insecure to expose the RHACS portal directly to external clients by using plain HTTP.
+====
+//TODO Add a link to the endpoint configuration topic as the recommended way to expose portal.
+
+To specify a HTTP endpoint you must use an `<endpoints_spec>`.
+
+`<endpoints_spec>` is a comma-separated list of single endpoint specifications in the form of `<type>@<addr>:<port>`, where:
+
+* `type` is `grpc` or `http`. Using `http` as type works in most use cases.
+For advanced use cases, you can either use `grpc` or omit its value.
+If you omit the value for `type`, you can configure two endpoints in your proxy, one for gRPC and the other for HTTP.
+Both these endpoints point to the same exposed HTTP port on Central.
+However, most proxies do not support carrying both gRPC and HTTP traffic on the same external port.
+* `addr` is the IP address on which you are planning to expose Central.
+You can omit this, or use `localhost` or `127.0.0.1` if you need an HTTP endpoint which is only accessible by using port-forwarding.
+* `port` is the port number on which you are going to expose Central.
```

Comment:
Per IBM style guide, you are allowed to end in a preposition if it makes the sentence less awkward. I'd consider simplifying to just "is the port to expose Central on." or something like that.

Same for addr if you do